README.md

```diff
@@ -40,11 +39,11 @@ modules, eliminating the need for headers and the code duplication they entail.
 
 To learn more about the programming language, visit [swift.org](https://swift.org/documentation/).
 
-- [Contributing to Swift](#contributing-to-swift)
-- [Getting Started](#getting-started)
-  - [Swift Toolchains](#swift-toolchains)
-  - [Build Failures](#build-failures)
-- [Learning More](#learning-more)
+### Table Of Contents
+1. [Contributing to Swift](https://github.com/apple/swift#contributing-to-swift)
```

Comment:
Is there a reason you’ve changed these links?

Reply:
Ohh thats just how I always do I prob should just do the shortcut way and do something like this 

[Getting Started](#getting-started). Thx for lmk.

Reply:
```md
Getting Started](#getting-started)
```

Reply:
i meant like this ```md
Getting Started](#getting-started)
```